specification/recoveryservicesbackup/resource-manager/readme.python.md

```diff
@@ -54,5 +54,201 @@ directive:
     where: $.definitions.FeatureSupportRequest
     transform: >
         $['required'] = ['featureType'];
-
-```
+  - from: swagger-document
+    where: $.definitions.OperationWorkerResponse.properties.statusCode
+    transform: >
+         $['x-ms-enum'] = {
+            "name": "HttpStatusCode",
+            "modelAsString": false,
+            "values": [
+                {
+                  'name': 'CONTINUE_ENUM',
```

Comment:
Fix https://github.com/Azure/azure-sdk-tools/issues/9542#issuecomment-2561649304.
After codegen upgrade, there is no additional suffix `ENUM` added for `CONTINUE`. However, to keep backward compatibility, we have to add directives to make the enum key name same as before.